### ISSUE TITLE
Report-a comic about the lost archive

### ISSUE THREAD

**Issue by Reiquard** [jptrrs/HumanResources#138]
There is a pawn **Laila** and she has a knowledge of **[Colored lights]** while the colony does not
![1](https://user-images.githubusercontent.com/62316596/107960908-422e6580-6fb6-11eb-9789-158fb644b44f.png)

I ordered her to document this knowledge
![2](https://user-images.githubusercontent.com/62316596/107960940-4ce8fa80-6fb6-11eb-9e4f-5549052b7303.png)

When she was done _(which was pretty fast, to be honest)_,
the network server status string was `Archived technologies: 1/8`
and the tooltip of the cloud near the research node was `Archived in our database.`
![3](https://user-images.githubusercontent.com/62316596/107961100-7f92f300-6fb6-11eb-9653-e8258ebe1409.png)

Then I saved the game.

After restarting RimWorld and loading this save,
the network server status string was `Empty`
and there was a book (not a cloud) near the research node and its tooltip was "Archived in our library."
![4](https://user-images.githubusercontent.com/62316596/107961200-9fc2b200-6fb6-11eb-988c-43665a2c2cd4.png)

But our library was completely empty
![5](https://user-images.githubusercontent.com/62316596/107963887-c1716880-6fb9-11eb-8f19-ef7228d956fa.png)

Everything described above happened in the newly created "Crashlanded" scenario.

By the way, if I RMB-click on the [Study Desk] with only [Study technology] bill active while Laila with "documenting" assignment is selected, then I will see this:
![image](https://user-images.githubusercontent.com/62316596/107978672-41093280-6fce-11eb-88e2-3ef0a6c7ebcf.png)

activeMods
      - brrainz.harmony
      - ludeon.rimworld
      - ludeon.rimworld.royalty
      - unlimitedhugs.hugslib
      - fluffy.modmanager
      - vinalx.researchpalforked
      - jpt.humanresources   _(latest master)_
/activeMods

**Comment by jptrrs** [comment]
The clearest bug report ever! :-)
Fixed.

**Comment by Reiquard** [comment]
It seems like nothing has changed regarding this issue with the latest master version.

Along with testing this, I noticed two things:

 - When **Laila** finished documenting, a message appeared:
`The knowledge for colored lights was added to our library`
Although she was documenting at a network terminal, and the technology was added to the database, not the library.

 - If I order **Laila** to document her knowledge (lets take the **spacer** level for surety) while a network server is not present on the map, I will get this JobFailReason:
`Nowhere to store the book when it's finished`
Although the books are limited to the industrial level.
